src/inverseRobot.js

```diff
@@ -24,7 +24,25 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+
+  if (robot === {}) {
+    return robot;
+  }
+
+  const ret = {};
+
+  Object.keys(robot).forEach(key => {
```

Comment:
try to solve this task without Object.values and Object.keys, you don't really need them here